webapp/views/index.erb

```diff
@@ -0,0 +1,46 @@
+<html>
+<head>
+<title>Continue benchmarking microsite</title>
+<link rel="stylesheet" type="text/css" href="css/style.css">
+<script src="//ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
+<script src="js/Chart.js"></script>
+</head>
+<body>
+<h1>Continious benchmarking microsite</h1>
+
+<p>
+Taleggio airedale mozzarella. Cheesy feet mascarpone mozzarella ricotta stilton paneer fondue cauliflower cheese. Cheese triangles cheesecake fondue caerphilly chalk and cheese cheddar cream cheese cut the cheese. Pepper jack boursin halloumi camembert de normandie roquefort blue castello paneer goat. Halloumi dolcelatte babybel stinking bishop.
```

Comment:
I'd suggest either an empty <p></p> tag or replacing this with content. I know this is early stage but doesn't add anything to the user